app/auth/GatehouseUserInfoResponseParser.scala

```diff
@@ -0,0 +1,35 @@
+package auth
+
+import com.gu.identity.auth.UserInfoResponseParser.{identityId, primaryEmailAddress}
+import com.gu.identity.auth.{CirceJsonParser, IdentityClaims, JsonString, UserInfoResponseParser, ValidationError}
+
+case class GatehouseIdentityClaims(
+    primaryEmailAddress: String,
+    identityId: String,
+    oktaId: String
+) extends IdentityClaims
```

Comment:
Where does this get used?

Reply:
I can't because of the bug in the identity library we were discussing in the chat. I'll fix it so that I can use this